react-portal-employees/src/components/about.tsx

```diff
@@ -1,27 +1,250 @@
+                            labelId="demo-simple-select-label"
+                            id="demo-simple-select"
+                            label="English">
+                            <MenuItem value={10}>Novice</MenuItem>
+                            <MenuItem value={20}>Professional</MenuItem>
+                        </Select>
+                    </FormControl>
+                    <FormControl fullWidth sx={{
+                        marginBlock: 2,
+                    }}>
+                        <InputLabel id="demo-simple-select-label">Leadership</InputLabel>
+                        <Select
+                            labelId="demo-simple-select-label"
+                            id="demo-simple-select"
+                            label="Leadership">
+                            <MenuItem value={10}>Novice</MenuItem>
+                            <MenuItem value={20}>Professional</MenuItem>
+                        </Select>
+                    </FormControl>
+                    <FormControl fullWidth sx={{
+                        marginBlock: 2,
+                    }}>
+                        <InputLabel id="demo-simple-select-label">Communication Skills</InputLabel>
+                        <Select
+                            labelId="demo-simple-select-label"
+                            id="demo-simple-select"
+                            label="Communication">
+                            <MenuItem value={10}>Novice</MenuItem>
+                            <MenuItem value={20}>Professional</MenuItem>
+                        </Select>
+                    </FormControl>
+                </FormGroup>
+
+                <Typography>
+                    Hard Skills
+                </Typography>
+
+                <FormGroup>
+                    <FormControl fullWidth sx={{
+                        marginBlock: 2,
+                    }}>
+                        <InputLabel id="demo-simple-select-label">C#</InputLabel>
+                        <Select
+                            labelId="demo-simple-select-label"
+                            id="demo-simple-select"
+                            label="C#">
+                            <MenuItem value={10}>Novice</MenuItem>
+                            <MenuItem value={20}>Advanced Beginner</MenuItem>
+                            <MenuItem value={30}>Competent</MenuItem>
+                            <MenuItem value={40}>Proficient</MenuItem>
+                            <MenuItem value={50}>Expert</MenuItem>
+                        </Select>
+                    </FormControl>
+                    <FormControl fullWidth sx={{
+                        marginBlock: 2,
+                    }}>
+                        <InputLabel id="demo-simple-select-label">SQL</InputLabel>
+                        <Select
+                            labelId="demo-simple-select-label"
+                            id="demo-simple-select"
+                            label="SQL">

```

Comment:
Actually, we will fetch all skills and possible skill values, so add a controller with mocked values and keep this in mind when you will be adding it